Developers/Writing_Good_Commits.rst

```diff
@@ -1,6 +1,31 @@
 How To Write a Good Commit Message
 ==================================
 
+Quick reference
+---------------
+
+Example of a good commit:
+
+::
+
+    setup.py: Change bears' entrypoint
+
+    This entrypoint ensures that coala discovers
+    the bears correctly.
+    It helps not writing more functions inside
+    ``coalib`` for this.
+
+    Fixes https://github.com/coala/coala/issues/5861
+
+- `setup.py: Change bears' entrypoint`: Describe the change in
+   maximum of 50 characters.
+
+- `This entrypoint.. ..for this`: Describe the reasoning of your changes
+   in maximum of 72 characters.
+
+- `Fixes https://github.com/coala/coala/issues/5861`: Mention the URL
+   of the issue it closes or fixes.
+
```

Comment:
@Makman2 What's the point if we put it below the abstract. If the user reach at the end of the page means(most probably) that he has read the section and he doesn't need anymore of this reference.